subprojects/packagefiles/catch2/meson.build

```diff
@@ -1,9 +1,10 @@
-project('catch2',
-    'cpp',
-    version : '2.13.8',
-    license : 'Boost'
-)
+project(
+  'catch2',
+  'cpp',
+  version : '3.1.0',
+  license: 'Boost 1.0')
```

Comment:
Once we are updating the license to make it more accurate, we might as well use the SPDX identifier,  `BSL-1.0`.